src/dali/plugin/input/rest/coinbase_pro.py

```diff
@@ -320,60 +321,60 @@ def _process_fills(self, transaction: Any, in_transaction_list: List[InTransacti
             self.__append_transaction(
                 cast(List[AbstractTransaction], out_transaction_list),
                 OutTransaction(
-                    self.__COINBASE_PRO,
-                    unique_id,
-                    json.dumps(transaction),
-                    fill[_CREATED_AT],
-                    from_currency,
-                    self.__COINBASE_PRO,
-                    self.account_holder,
-                    "Sell",
-                    str(from_currency_price),
-                    str(from_currency_size),
-                    "0",
-                    None,
-                    None,
-                    None,
-                    f"Sell side of conversion: {from_currency_size:.8f} {from_currency} -> {to_currency_size:.8f} {to_currency}",
+                    plugin=self.__COINBASE_PRO,
+                    unique_id=unique_id,
+                    raw_data=json.dumps(transaction),
+                    timestamp=fill[_CREATED_AT],
+                    asset=from_currency,
+                    exchange=self.__COINBASE_PRO,
+                    holder=self.account_holder,
+                    transaction_type="Sell",
+                    spot_price=str(from_currency_price),
+                    crypto_out_no_fee=str(from_currency_size),
+                    crypto_fee="0",
+                    crypto_out_with_fee=None,
+                    fiat_out_no_fee=None,
+                    fiat_fee=None,
+                    notes=f"Sell side of conversion: {from_currency_size:.8f} {from_currency} -> {to_currency_size:.8f} {to_currency}",
```

Comment:
best practice in python to use keyword args calling style to avoid passing in the wrong argument with the wrong index especially for functions that takes a long list of arguments.